hw5-10_to_do_list/src/ts/Input.ts

```diff
@@ -0,0 +1,16 @@
+import Component from "./Component";
+
+export default class Input extends Component {
+    element: HTMLInputElement
+    constructor(props: {type: string, cssClass?: string, placeholder?: string}) {
+        super({ ...props, elementType: 'input' })
```

Comment:
In all such places I see this error:
![Screenshot 2023-04-28 at 21 10 16](https://user-images.githubusercontent.com/70685561/235212680-90b7721b-c522-4f24-ba33-09fbc2c8bcfb.png)
Probably you have weak rules in tsconfig and you don't catch it.  It is okey, no complains, but in real projects you need more strict check and props conversion between class and superclass.